spark-etl/src/main/scala/geotrellis/spark/etl/accumulo/SpatialAccumuloInput.scala

```diff
@@ -20,7 +20,7 @@ class SpatialAccumuloInput extends AccumuloInput {
       case Some(extent) => catalog.query[SpatialKey](id).where(Intersects(extent)).toRDD
       case None => catalog.read[SpatialKey](id)
     }
-    LayoutLevel(id.zoom, rdd.metaData.tileLayout) -> rdd.asInstanceOf[RasterRDD[K]]
+    id.zoom -> rdd.asInstanceOf[RasterRDD[K]]
```

Comment:
I've been avoiding using the `->` notation, because the `@inline` usually fails and that's where all the compiler warnings get when we run with the `-optimized` flag.